includes/admin/shortcodes/shortcode-give-donation-grid.php

```diff
@@ -55,6 +55,30 @@ public function define_fields() {
 				'tooltip'     => esc_attr__( 'Enter a comma-separated list of form IDs to exclude those from the grid.', 'give' ),
 				'placeholder' => esc_html__( 'Excluded Forms', 'give' )
 			),
+			array(
+				'type'    => 'listbox',
+				'name'    => 'orderby',
+				'label'   => esc_attr__( 'Order By:', 'give' ),
+				'tooltip' => esc_attr__( 'Different parameter to set the order for the forms display in the form grid.', 'give' ),
+				'options' => array(
+					'date'             => esc_html__( 'Date Created', 'give' ),
+					'name'             => esc_html__( 'Form Name', 'give' ),
+					'amount_donated'   => esc_html__( 'Amount Donated', 'give' ),
+					'number_donations' => esc_html__( 'Number of Donations', 'give' ),
+					'menu_order'       => esc_html__( 'Menu Order', 'give' ),
```

Comment:
@kakshak @kevinwhoffman `menu_order` mostly used for pages and it will not work for the custom post type. Can we remove that because `post__in` will fulfill or requirement?